docs/Database.md

```diff
@@ -5,6 +5,46 @@ The `CCT` and `MyGordon` databases exist in:
 - `admintrainsql.gordon.edu` - The development/testing database server
 - `adminprodsql.gordon.edu` - The live production database server.
 
+### Updating context tables (_new columns, tables, procedures..._)
+
+SQL Table models are handled in `360api` by Entity Framework Power Tools (`efpt`). To script in new tables do the following:
+
+0) This _might_ need to be done: On to the top tab of Visual Studio
+   - click `tools` -> `Connect to Database...`.
+   - select `Microsoft SQL Server` -> `Continue`
+   - <img width="509" alt="image" src="https://github.com/gordon-cs/gordon-360-api/assets/78386128/344d27eb-0882-44e6-812d-7ea4c7eb8be9">
+
+     - select the appropriate Database Server (listed above)
+     - select the appropriate database (CCT, MyGordon...)
+     - press `ok`
+   - It is highly suggested to commit/stash your existing changes before doing an `efpt` edit.

```

Comment:
This is not necessary, it's irrelevant to the EF Core connections.

Reply:
Maybe say that it's helpful to work in a clean branch, so in step 2 it's easier to see what has changed and what might need to be reverted.